src/main/java/de/kaktushose/nrtv/discord/core/Bootstrapper.java

```diff
@@ -62,6 +62,42 @@ public static void main(String[] args) {
                     bot.getJda().getPresence().setActivity(Activity.playing("Farming Simulator 19"));
                     break;
                 case 3:
+                    counter.set(counter.get() + 1);
+                    bot.getJda().getPresence().setActivity(Activity.playing("Notruf 112 2"));
```

Comment:
suggestion: change to `Notruf 2` as this is the informal title and is imo better readable